presto-druid/pom.xml

```diff
@@ -373,4 +326,18 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.basepom.maven</groupId>
+                <artifactId>duplicate-finder-maven-plugin</artifactId>
+                <configuration>
+                    <ignoredClassPatterns>
+                        <ignoredClassPattern>it.unimi.dsi.fastutil.*</ignoredClassPattern>
```

Comment:
Why do we have duplicates of fastutil?

Reply:
With the upgrade of Druid to version 30.0.1, fastutil as a transitive dependency was updated from version 5.2.2 to 5.2.4 and is now split into separate artifacts (fastutil-core, fastutil-extra, and fastutil). Although Fastutil is excluded in the root pom through the druid-processing dependency, some files from fastutil-core were still being included which raise conflicts. To bypass the duplicate warnings, I added fastutil to the ignoredClassPattern in the duplicate-finder-maven-plugin. However, this was not a feasible solution, and to properly resolve the issue, I have now explicitly excluded fastutil-core from druid-processing to maintain consistency across the project.